### PR TITLE
README.md: update section about repo

### PR DESCRIPTION
**Pull request by jlebon**
Tweak the section talking about how this repo works to reflect the
latest state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,16 +22,18 @@ and [tooling](https://github.com/coreos/fedora-coreos-tracker/blob/main/stream-t
...
-  imported from `bodhi-updates` to `testing-devel`.
-  Overrides (`manifest-lock.overrides.*`) are manually
-  curated.
+and to `testing` and eventually `stable` through usual
+promotion), with the following exceptions:
+- `manifest.yaml`: contains the stream's name, yum repos
+  used during composes, and the `releasever`.
+- lockfiles (`manifest-lock.*` files): lockfiles are pushed
+  to `testing-devel` and `next-devel` by
+  [the `bump-lockfile` job](https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/bump-lockfile.Jenkinsfile).
```
Comment (dustymabe) [inline README.md]: There's some nuance here. They get pushed to `testing-devel` and `next-devel` through bump-lockfile, but then they get promoted to the other branches through normal promotion.

File: README.md [hunk 2]
```diff
@@ -22,16 +22,19 @@ and [tooling](https://github.com/coreos/fedora-coreos-tracker/blob/main/stream-t
...
+promotion), with the following exceptions:
+- `manifest.yaml`: contains the stream's name, yum repos
+  used during composes, and the `releasever`.
+- lockfiles (`manifest-lock.*` files): on `testing-devel`
+  and `next-devel`, lockfiles are pushed by
+  [the `bump-lockfile` job](https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/bump-lockfile.Jenkinsfile).
+  Production streams receive them as part of usual
+  promotion. Overrides (`manifest-lock.overrides.*`) are
+  managed independently with the help of some GitHub Actions
+  (see sections below).
```
Comment (dustymabe) [inline README.md:37]: Still a bit more nuance.. Here's my stab at adding some more clarity:

```suggestion
- lockfiles (`manifest-lock.$ARCH.yaml` files): on `testing-devel`
  and `next-devel`, lockfiles are pushed by
  [the `bump-lockfile` job](https://github.com/coreos/fedora-coreos-pipeline/blob/main/jobs/bump-lockfile.Jenkinsfile).
  Production streams receive them as part of usual
  promotion.
- overrides lockfile (`manifest-lock.overrides.yaml`):
  On `testing-devel` and `next-devel` this file is managed
  by hand or updated by some helpful GitHub Actions (see
  sections below). For other branches this file is promoted
  with normal promotion, but can also be updated independently
  if a need arises for a change in the package set for an ad-hoc
  release.
```

Let me know if you think it helps.

Comment (jlebon) [reply README.md:37]: Hmm maybe. I wonder if we should fold some of those details into lower sections instead and keep this bit lighter. Maybe let's just get this in and iterate there? Seems like a good improvement overall to what we currently have.

Comment (dustymabe) [reply README.md:37]: sounds good

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**